sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/hello_pipeline_job_with_registries.yml

```diff
@@ -8,7 +8,6 @@ inputs:
 jobs:
   a:
     component: azureml://registries/testFeed/components/my_hello_world_asset_2/versions/1
-    command: echo hello ${{inputs.hello_string}}
```

Comment:
@nancy-mejia could you please share the context why you add command here? To my understanding, command overwirte for a remote component is forbidden by design.

Reply:
Given you're on vacation, I'll merge the PR first and let's follow-up after you're back.